README.md

```diff
@@ -43,6 +43,7 @@ USAGE
 # Commands
 <!-- commands -->
 * [`chectl autocomplete [SHELL]`](#chectl-autocomplete-shell)
+* [`chectl devfile:generate`](#chectl-devfilegenerate)
```

Comment:
It's a minor comment but it would be better to leave `help` command at the beginning.

Reply:
@sleshchenko the README is generated automatically by the oclif framework. The flags are ordered alphabetically